extensions/query-store/src/test/utils.test.ts

```diff
@@ -0,0 +1,142 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as azdata from 'azdata';
+import * as should from 'should';
+import { createOneComponentFlexContainer, createTwoComponentFlexContainer, createVerticalSplitView } from '../common/utils';
+
+interface TestContext {
+	view: azdata.ModelView;
+	component: azdata.Component
+}
+
+function createViewContext(): TestContext {
```

Comment:
can this be moved out into a separate file, like `testUtils.ts` in sql projects, so that this file only has unit tests?

Reply:
Updated and moved utility methods to testUtils file.